File: src/AspNetCore/AspNetCore.Vue.Vuetify1.ElectronNet/Controllers/HomeController.cs
using Microsoft.AspNetCore.Mvc;
using System.Diagnostics;

namespace AspNetCore.Vue.Vuetify1.ElectronNet.Controllers {

    /// <summary> Home Controller. </summary>
    public class HomeController : Controller {

        /// <summary> Index Page. </summary>
        /// <returns> The view. </returns>
        public IActionResult Index() {
            return View();
        }

        /// <summary> Error Page. </summary>
        /// <returns> The view. </returns>
        public IActionResult Error() {
            ViewData["RequestId"] = Activity.Current?.Id ?? HttpContext.TraceIdentifier;
            return View();
        }
    }
}

File: src/Python.Cef/PythonCef.Vue.Vuetify2/src/ServerApp/Controllers/HomeController.py
from flask import Blueprint, render_template, abort

HomeController = Blueprint('HomeController', __name__)


@HomeController.route('/')
def home():
    return render_template(
        'home.html',
        title='PythonFlask.Vue.Vuetify1 Template'
    )


#using System;
#using System.Collections.Generic;
#using System.Diagnostics;
#using System.Linq;
#using System.Threading.Tasks;
#using Microsoft.AspNetCore.Mvc;

#namespace AspNetCore.Vue.Vuetify1.Controllers
#{
#    public class HomeController : Controller
#    {
#        public IActionResult Index()
#        {
#            return View();
#        }

#        public IActionResult Error()
#        {
#            ViewData["RequestId"] = Activity.Current?.Id ?? HttpContext.TraceIdentifier;
#            return View();
#        }
#    }
#}
